pyomo/gdp/disjunct.py

```diff
@@ -158,15 +167,31 @@ def __init__(self, *args, **kwds):
 
 
 class IndexedDisjunct(Disjunct):
-    pass
+    #
+    # HACK: this should be implemented on ActiveIndexedComponent, but
+    # that will take time and a PEP
+    #
+    @property
+    def active(self):
+        return any(d.active for d in itervalues(self._data))
+
+    @active.setter
```

Comment:
@jsiirola, I'm confused: You can't set `active`, right? We added this in both IndexedDisjunct and IndexedDisjunction but it's not covered in the tests because I don't know how to call it...